kuma/static/js/interactive.js

```diff
@@ -7,6 +7,84 @@
+            var currentEntry = perfEntries[i];
+            if (
+                currentEntry.initiatorType &&
+                currentEntry.initiatorType === 'iframe' &&
+                currentEntry.name.indexOf(
+                    'interactive-examples.mdn.mozilla.net'
+                ) > -1
+            ) {
+                return perfEntries[i];
+            }
+        }
+    }
+
+    /* Ensure there is an iframe present and that performance is
+       supported before gathering performance metric */
+    if (iframe && performance !== undefined) {
+        document.addEventListener('readystatechange', function(event) {
+            if (event.target.readyState === 'complete') {
+                var interactiveExamplesPerfEntry = {};
+                var perfEntries = performance.getEntriesByType('resource');
+                var mainFetchStart = 0;
+                var iframeFetchStart = 0;
+                var iframeFetchStartSinceUnixEpoch = 0;
+
+                if (perfEntries === undefined || perfEntries.length <= 0) {
+                    console.info('No performance entries was returned');
+                    return;
+                }
+
+                interactiveExamplesPerfEntry = getInteractiveExamplesPerfEntry(
+                    perfEntries
+                );
+
+                mainFetchStart = performance.timing.fetchStart;
+                iframeFetchStart = Math.round(
+                    interactiveExamplesPerfEntry.fetchStart
+                );
+                iframeFetchStartSinceUnixEpoch =
+                    mainFetchStart + iframeFetchStart;
+
+                // var iframeLoadTime = interactiveExamplesPerfEntry.responseEnd - interactiveExamplesPerfEntry.startTime;
+                // ga('send', {
+                //     'eventCategory' : 'Interactive Examples',
+                //     'eventAction'   : 'iFrame load time',
+                //     'eventLabel'    : 'Performance Events',
+                //     'eventValue'    : (iframeLoadTime / 1000),
+                //     'hitType': 'timing'
+                // });
+                console.log(
+                    (interactiveExamplesPerfEntry.responseEnd -
+                        interactiveExamplesPerfEntry.startTime) /
+                        1000
+                );
+
+                // var timeToIframeFetchStart = new Date(iframeFetchStartSinceUnixEpoch).getSeconds() -
+                //    new Date(mainFetchStart).getSeconds()
+                // ga('send', {
+                //     'eventCategory' : 'Interactive Examples',
+                //     'eventAction'   : 'Time to iframe fetch start',
+                //     'eventLabel'    : 'Performance Events',
+                //     'eventValue'    : timeToIframeFetchStart,
+                //     'hitType': 'timing'
+                // });
+                console.log(
+                    'Time in seconds between fetch start of main page, and fetch start of iframe',
```

Comment:
Both of these metrics are accurate for what they measure, but we're still missing the full picture of how long it takes the iframe to be usable. 

For example, I loaded https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach up on a simulated slow 3G connection.

It took 6569ms for the iframe to be fetched, and another 2435 for it's response to end. So if we looked at total time, we end up at 9004ms.

But, this is what the screen looks like at that point:

![screen shot 2018-06-08 at 10 56 50 am](https://user-images.githubusercontent.com/66536/41168218-eb682f5e-6b0a-11e8-9e67-750edc2d1b99.png)

It takes another 10 seconds or so before it's load event has completed and we get a fully visible and usable iframe.

So I think we probably need to post iframe's `loadEventEnd` timestamp back to the parent window for this calculation to get the full picture.

Reply:
@tkadlec So, just the time it took for the iframe to completely load correct? We are not adding it to anything, or subtracting. Just the raw number in milliseconds.